docs/reference/dlm/index.asciidoc

```diff
@@ -0,0 +1,58 @@
+[role="xpack"]
+[[data-lifecycle-management]]
+== {dlm-init}: Manage your data lifecycle
+
+preview::[]
+
+The goal of {dlm} ({dlm-init}) is to enable you to automatically manage data streams
+according to your retention requirements. For example, you could use {dlm-init} to:
```

Comment:
`automatically manage data streams according to your retention requirements, without having to worry about...` complicated low-level configuration? what happens which each individual index? I'm not really sure. Just seems like this sentence undersells DLM :)

Reply:
I see what you mean. I do not want to mention indices here, or other implementation details. If we have new users that do not know ILM then this comparison will not be relevant. I can stress a bit more that it's "easy" and leave it at that.

In follow up documentation that would explain better the differences between ILM and DLM this comparison would be more appropriate. What do you think?

Reply:
I also paused on this sentence (in today's form where it ends with "without having to worry." I think this still needs some rewording as in this form I'm left thinking "worry about what?". I think what we mean is "without having to worry about internal implementation details." or "without having to consider as many details as with ILM."?

Reply:
I see, it was meant to be that it will just work :p . I will remove it. Let's leave comparisons for later.